plugins/cinema4d/import_cast.pyp

```diff
@@ -173,6 +177,7 @@ def importModelNode(doc, node, model, path):
         for i in range(mesh.UVLayerCount()):
             uvTag = c4d.UVWTag(facesCount)
             uvBuffer = mesh.VertexUVLayerBuffer(0)
+
             uvUnpacked = unpack_list([(uvBuffer[x * 2], 1.0 - uvBuffer[(x * 2) + 1]) for x in faces])
```

Comment:
Can we get rid of unpack_list? It should not be needed.

Reply:
I had some big issues as I tried to remove it, but I'll get back to it and try again. 
Here are the results I had in my tries:
![image](https://github.com/user-attachments/assets/665c9184-756d-4156-bef8-30c02335ced2)
![image](https://github.com/user-attachments/assets/a760f52e-09b1-4413-8528-92ff1e82052d)